packages/ui/stories/molecules/Button.stories.tsx

```diff
@@ -33,31 +33,63 @@ storiesOf('ui/molecules/Button', module).add(
   },
   {
     argTypes: {
-      title: {
-        type: 'string',
-        defaultValue: 'Press me',
-        control: {type: 'text'},
-      },
       color: {
         options: Object.keys(lightTheme.colors),
         mapping: lightTheme.colors,
         control: {
           type: 'select',
-          labels: {
-            primary: 'Primary',
-            secondary: 'Secondary',
-          },
         },
       },
       disabled: {
         type: 'boolean',
         defaultValue: false,
         control: {type: 'boolean'},
       },
+      FontAwesome5: {
+        type: 'boolean',
+        defaultValue: true,
+        control: {type: 'boolean'},
+      },
+      iconName: {
+        type: 'string',
+        defaultValue: 'check',
+        control: {type: 'text'},
+      },
+      isNeutralBackground: {
+        type: 'boolean',
+        defaultValue: false,
+        control: {type: 'boolean'},
+      },
+      onDisabledPress: {
+        action: 'clicked',
+        table: {disable: true},
+      },
       onPress: {
         action: 'clicked',
         table: {disable: true},
       },
+      style: {
+        control: {
+          type: 'object',
+        },
+        defaultValue: {},
+      },
+      styleTxt: {
+        control: {
+          type: 'object',
+        },
+        defaultValue: {},
+      },
```

Comment:
You do not need to add style in storybook, it's do complicated to fill in when you are on the story, the editor is not practical